CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+<!-- omit in toc -->
+# Contributing to Capgo
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+### Running tests localy
+
+This project uses a custom test runner located in [tests_backend](https://github.com/Cap-go/capgo/tree/main/tests_backend).
+There exists some requirements to run the tests:
+ * Have minio running (see `tests_backend/gh_actions` for referance) For testing the CLI this is not enough, you have to set up the `MINIO_URL` variable. In linux you can use 
+ ```sh
+ export MINIO_URL=$(docker inspect minio1 | grep 'Gateway' | head -n 1 | sed -e 's/            "Gateway": "//g' | sed -e 's/",//g')
+ ```
+ * Have redis running (see `tests_backend/gh_actions` for referance, only if `USE_LOCAL_REDIS` env variable is set to true, tests run fine without it)
+ * Having the `UPSTASH_TOKEN` and `UPSTASH_URL` env variables set (Only if you want to test upstash, tests run fine without it)
+ * Having `node`, `npx`, `pnpm` installed (Only for CLI tests)
+ * Having the [supabase cli](https://supabase.com/docs/guides/cli) installed
+ * Having a running supabase (`supabase start`)
+
+The tests can be run with the following commands:
+ * `pnpm test:backend` (backend only)
```

Comment:
we could add a `pnpm test:setup` to do stuff like start supabase/up redis (if we need to up it)

Reply:
I do not think this is a great idea, we are already doing this from the [CI/CD](https://github.com/Cap-go/github-bot/blob/8d796aa830627b7d70e169b02de71f1bda86aec4/.github/workflows/test_cli.yml#L109) and this script in `gh_actions` sets minio in a very temporary way (Making the data folder in the `/tmp` making the entire data disappear on a system reset).

Also again: This script in `gh_actions` is only for linux as a reference and it does not work on windows.

Also by default only minio and supabase is required and the tests will run fine without redis
![image](https://github.com/Cap-go/capgo/assets/50914789/39d49850-99d8-4025-ae46-c514920000a8)

Reply:
IDK if we currently support Windows as a development OS, but I don't think it's required to support Windows to have stuff on node scripts if it's not something that we'll usually run locally to test against new changes; feel free to solve this issue as it is